tests/unit/test-util/test.artifacts.js

```diff
@@ -55,4 +55,74 @@ describe('prepareArtifactsDir', () => {
     }
   ));
 
+  it('throws an UsageError when it lacks permissions to stat the directory',
+    () => withTempDir(
+    function(tmpDir) {
```

Comment:
nit: I'd like to see this block (along with those of similar test cases) indented two spaces to show that it's a continuation of the call started above.

When we do https://github.com/mozilla/web-ext/issues/755 we can get rid of `withTempDir` ! It's annoying, sorry.